### PR TITLE
feat: embedding attribute semantic conventions and llamaindex implementation

### PR DESCRIPTION
**Pull request by axiomofjoy**

### CODE REVIEW

File: src/phoenix/trace/semantic_conventions.py [hunk 1]
```diff
@@ -61,6 +61,19 @@ def _missing_(cls, v: Any) -> Optional["MimeType"]:
...
 
+EMBEDDINGS_VECTOR = "embedding.vector"
+"""
+The embedding vector.
+"""
+EMBEDDINGS_TEXT = "embedding.text"
+"""
+In the case of textual data, the text represented by the embedding vector.
+"""
+EMBEDDINGS_MODEL_NAME = "embedding.model_name"
```
Comment (RogerHYang) [inline src/phoenix/trace/semantic_conventions.py:72]: are these plural or singular?
```suggestion
EMBEDDING_VECTOR = "embedding.vector"
"""
The embedding vector.
"""
EMBEDDING_TEXT = "embedding.text"
"""
In the case of textual data, the text represented by the embedding vector.
"""
EMBEDDING_MODEL_NAME = "embedding.model_name"
```

File: src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py [hunk 2]
```diff
@@ -53,11 +55,12 @@ def payload_to_semantic_attributes(payload: Dict[str, Any]) -> Dict[str, Any]:
     """
     attributes = {}
 
+    if EventPayload.CHUNKS in payload and EventPayload.EMBEDDINGS in payload:
+        attributes[EMBEDDINGS_TEXT] = payload[EventPayload.CHUNKS]
+        attributes[EMBEDDINGS_VECTOR] = payload[EventPayload.EMBEDDINGS]
```
Comment (RogerHYang) [inline src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py:60]: are these plural or singular?

File: integration-tests/experimental/callbacks/test_llama_index_callback_handler.py [hunk 3]
```diff
@@ -68,6 +70,13 @@ def test_callback_handler_records_llm_message_attributes_for_query_engine(
     assert role == "user"
     assert query in message_text
 
+    span = next(span for span in tracer.span_buffer if span.span_kind == SpanKind.EMBEDDING)
+    embedding_texts = span.attributes["embedding.text"]
+    embedding_vectors = span.attributes["embedding.vector"]
```
Comment (RogerHYang) [inline integration-tests/experimental/callbacks/test_llama_index_callback_handler.py:75]: are these plural or singular?
```suggestion
    embedding_texts = span.attributes[EMBEDDING_TEXT]
    embedding_vectors = span.attributes[EMBEDDING_VECTOR]
```

Comment (axiomofjoy) [reply integration-tests/experimental/callbacks/test_llama_index_callback_handler.py]: addressed in [2fe71e5](https://github.com/Arize-ai/phoenix/pull/1169/commits/2fe71e55efc00378f6c7b82fa6e08905b0348bfb)

File: src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py (hunk 2 above)
Comment (axiomofjoy) [reply src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py]: addressed in [2fe71e5](https://github.com/Arize-ai/phoenix/pull/1169/commits/2fe71e55efc00378f6c7b82fa6e08905b0348bfb)

File: src/phoenix/trace/semantic_conventions.py (hunk 1 above)
Comment (axiomofjoy) [reply src/phoenix/trace/semantic_conventions.py]: addressed in [2fe71e5](https://github.com/Arize-ai/phoenix/pull/1169/commits/2fe71e55efc00378f6c7b82fa6e08905b0348bfb)

**Comment by axiomofjoy** [comment]
@RogerHYang Updated this PR to match the convention of a list of objects.

File: src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py [hunk 4]
```diff
@@ -53,11 +54,14 @@ def payload_to_semantic_attributes(payload: Dict[str, Any]) -> Dict[str, Any]:
     """
     attributes = {}
 
+    if EventPayload.CHUNKS in payload and EventPayload.EMBEDDINGS in payload:
+        attributes[EMBEDDING_EMBEDDINGS] = [
+            {"text": text, "vector": vector}
```
Comment (RogerHYang) [inline src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py:59]: let's make these keys (and in fact all keys) semantic conventions also
```suggestion
            {EMBEDDING_TEXT: text, EMBEDDING_VECTOR: vector}
```

File: src/phoenix/trace/semantic_conventions.py [hunk 5]
```diff
@@ -61,6 +61,15 @@ def _missing_(cls, v: Any) -> Optional["MimeType"]:
         return None if v else cls.TEXT
 
 
+EMBEDDING_EMBEDDINGS = "embedding.embeddings"
```
Comment (RogerHYang) [inline src/phoenix/trace/semantic_conventions.py:64]: may be sufficient to call this just embeddings?
```suggestion
EMBEDDINGS = "embeddings"
```

Comment (axiomofjoy) [reply src/phoenix/trace/semantic_conventions.py:64]: Where would the embedding model name and other metadata go in this case?

Comment (RogerHYang) [reply src/phoenix/trace/semantic_conventions.py:64]: they can go into the embedding object

Comment (RogerHYang) [reply src/phoenix/trace/semantic_conventions.py:64]: mostly because we're running out of words. we are using EMBEDDING in two places...

Comment (axiomofjoy) [reply src/phoenix/trace/semantic_conventions.py:64]: Certainly possible, but that lead to some redundancies.

```
[
    {
        "embedding": [...],
        "text": "text-0",
        "model_name": "text-embedding-ada-002"
    },
    {
        "embedding": [...],
        "text": "text-1",
        "model_name": "text-embedding-ada-002"
    }
]
```

File: src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py (hunk 4 above)
Comment (axiomofjoy) [reply src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py]: addressed in [31028c3](https://github.com/Arize-ai/phoenix/pull/1169/commits/31028c3ae0f7bf10266c5751bcf6606fde598f02)

**Pull request merged**